.github/workflows/bandit.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       security-events: write
     steps:
       - name: Perform Bandit Analysis
-        uses: PyCQA/bandit-action@8a1b30610f61f3f792fe7556e888c9d7dffa52de
+        uses: PyCQA/bandit-action@67a458d90fa11fb1463e91e7f4c8f068b5863c7f
```

Comment:
PLEASE DO NOT TRIGGER CI OR MERGE: This undoes the change from PR #911

Merging #913 will stop these automatic updates (until a better solution becomes available).